src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs

```diff
@@ -134,6 +135,43 @@ public void SendAsync_SlowServerRespondsAfterDefaultReceiveTimeout_ThrowsHttpReq
             }
         }
 
+        [OuterLoop]
+        [Theory]
+        //Take internationalized domain name 'Bangladesh' sample url from http://www.i18nguy.com/markup/idna-examples.html                     
+        [InlineData("http://\u09AC\u09BE\u0982\u09B2\u09BE\u09A6\u09C7\u09B6.icom.museum", "icom.museum")]
+        [InlineData("http://[::1234]", null)]
+        [InlineData("http://[::1234]:8080", null)]
+        [InlineData("http://127.0.0.1", null)]
+        [InlineData("http://www.microsoft.com", "www.microsoft.com")]        
+        public async Task ManualTest_IdnHostName(string requestUri, string requestHost)
```

Comment:
I think this manual test should be simplified.  The only test we really need to add is an end-to-end test that validates that  a request can be made to a server that uses Unicode characters in the DNS name.  The other test cases are really already covered by other IPv4 and IPv6 literal test cases in System.Net.Http/tests/FunctionalTests.

Also, then, this test would be more valuable as a test case in the System.Net.Http/tests/FunctionalTests folder so that it can be run, manually, across the different handlers.

This test should also be marked in a way so that it doesn't get run all the time in CI because then it will be trying to connect all the time to a server that doesn't really exist.

The error handling then for HttpRequestException doesn't need to be present then since the test is only validating a successful operation when run.

Even better, would be if we could use a third-party server on the public Internet that uses Unicode characters in the DNS names.  I'm sure that there are lots of them. Here is one that appears to be a international version of the Starbucks website.

Unicode:  스타벅스코리아.com
Punnycode: xn--oy2b35ckwhba574atvuzkc.com

So if you used that website, then the test could always be enabled and you would check for a successful connection.

Reply:
@davidsh ok i'll move test to System.Net.Http/tests/FunctionalTests with simple successful operation.

>This test should also be marked in a way so that it doesn't get run all the time in CI because then it will be trying to connect all the time to a server that doesn't really exist

Do you mean [Outerloop] or do you have some test sample to take a cue?

Reply:
If you use the website (international Starbucks) that I described above, then you can simply mark the test as [Outerloop].  It will still get run all the time during outerloop CI runs. By default, tests in CI only run innerloop.

However, if the website was something that can't normally be reached, then you would mark this test in some other way so that it never runs in innerloop nor outerloop.

I recommend you simply try to use the website I indicated and then the test can run in outerloop all the time.